cluster/points_writer.go

```diff
@@ -233,6 +237,9 @@ func (w *PointsWriter) WritePoints(p *WritePointsRequest) error {
 		}(shardMappings.Shards[shardID], p.Database, p.RetentionPolicy, points)
 	}
 
+	// Fire and forget points to Subscriber
+	go w.Subscriber.WritePoints(p)
```

Comment:
It might better to write this on a channel with a `default` case to prevent it from blocking on writing on the channel as well.  If this code is not async, a slow subscriber could block writes.  Something like:

```
select {
    case pointsC <- p:
        fmt.Println("sent points")
    default:
        fmt.Println("no points sent")
    }
```